Makefile

```diff
@@ -40,7 +40,7 @@ update: crd bindata
 # Generate CRDs from vendored and internal API specs.
 .PHONY: crd
 crd:
-	hack/update-generated-crd.sh
+	#hack/update-generated-crd.sh
```

Comment:
Is this needed for just the `00-custom-resource-definition.yaml` and `00-custom-resource-definition-internal.yaml` manifests?  Would it make more sense to add the `capability.openshift.io/name: Ingress` annotation in openshift/api?

Reply:
Yes, it would. This PR was created according to [how to implement a new capability](https://github.com/openshift/enhancements/blob/master/enhancements/installer/component-selection.md#how-to-implement-a-new-capability) to check the CI presubmit behavior when `Ingress` capability is unknown. That's why I had to comment this line to not get the overrides from the api repository.
The rest of the manifests still make sense, so the PR will live on, just this comment will have to be removed once the api repository is updated.